src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1979,6 +1984,27 @@ def SetCheckMatchingFabric(self, check: bool):
             lambda: self._dmLib.pychip_DeviceController_SetCheckMatchingFabric(check)
         ).raise_on_error()
 
+    def SetIcdRegistrationParameters(self, symmetricKey: bytes = None, checkInNodeId: int = 0, monitoredSubject: int = 0, stayActiveMs: int = 0):
+        if symmetricKey is not None:
+            if len(symmetricKey) != 16:
+                raise ValueError("symmetricKey should be 16 bytes")
+        if not checkInNodeId:
```

Comment:
should these default to None and be Optional[int] instead of 0 to distingush between set/not-set?

Reply:
Done